lms/djangoapps/instructor/management/commands/dump_grades.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/python
-"""
```

Comment:
Ah, my apologies.  This grading command is not one we run.  I believe this was replaced by the celery task that builds a grade report.